codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/AssembleMiddlewareStack.java

```diff
@@ -0,0 +1,145 @@
+package software.amazon.smithy.aws.go.codegen;
+
+import java.util.List;
+import software.amazon.smithy.aws.traits.auth.SigV4Trait;
+import software.amazon.smithy.aws.traits.auth.UnsignedPayloadTrait;
+import software.amazon.smithy.codegen.core.Symbol;
+import software.amazon.smithy.go.codegen.SmithyGoDependency;
+import software.amazon.smithy.go.codegen.SymbolUtils;
+import software.amazon.smithy.go.codegen.integration.GoIntegration;
+import software.amazon.smithy.go.codegen.integration.ProtocolGenerator;
+import software.amazon.smithy.go.codegen.integration.RuntimeClientPlugin;
+import software.amazon.smithy.model.traits.AuthTrait;
+import software.amazon.smithy.utils.ListUtils;
+
+public class AssembleMiddlewareStack implements GoIntegration{
+    @Override
+    public List<RuntimeClientPlugin> getClientPlugins() {
+        Symbol middlewareStepAfter = SymbolUtils.createValueSymbolBuilder(
+                "After", SmithyGoDependency.SMITHY_MIDDLEWARE
+        ).build();
+
+        Symbol middlewareStepBefore = SymbolUtils.createValueSymbolBuilder(
+                "Before", SmithyGoDependency.SMITHY_MIDDLEWARE
+        ).build();
+
+        Symbol RequestInvocationIDMiddleware = SymbolUtils.createValueSymbolBuilder(
+                "RequestInvocationIDMiddleware", AwsGoDependency.AWS_MIDDLEWARE
+        ).build();
+
+        Symbol attemptClockSkewMiddleware = SymbolUtils.createValueSymbolBuilder(
+                "AttemptClockSkewMiddleware", AwsGoDependency.AWS_MIDDLEWARE
```

Comment:
Looks like the Smithy or AWS middleware pkgs, need to define an alias because these are not being imported correct. Both are being imported as the "middleware" package which is causing the generated code to only include one of the import paths. an import alias on one of the two should fix the issue.